examples/eks_cluster_assumerole/main.tf

```diff
@@ -12,6 +12,7 @@ provider "helm" {
 
 provider "castai" {
   api_token = var.castai_api_token
+  api_url   = var.castai_api_url
```

Comment:
`castai_api_url` is not defined in variables.tf

Reply:
Damn, did not push it along with me local changes, fixed. Thanks ;)